code/modules/surgery/bones.dm

```diff
@@ -51,6 +51,12 @@
 		TOOL_BONEGEL = 100,
 		TOOL_SCREWDRIVER = 90
 	)
+
+	preop_sound = list(
+		/obj/item/bonegel = 'sound/surgery/organ1.ogg',
+		/obj/item/screwdriver = 'sound/items/screwdriver.ogg',
```

Comment:
These lists should be able to take tool behaviors *or* items as keys, similar to how `allowed_tools` works. We'd be missing sound effects on tools that can perform the surgery due to having the correct tool type, but that are not specified in the list itself. You can probably steal the logic from how `allowed_tools` gets checked.

Reply:
Listing a path of /obj/item/ = 'soundpathhere' works as a generic sound that'll play regardless of the item used. I did this for some of them but not all, could fix that if you'd like. Extra checks shouldn't be needed.

Reply:
I would still ask that it use tool behaviors for consistency with allowed tools when tools are specified. Having an /obj/item option should only really be needed if there's such an option in allowed tools or for something like the IMS imo.

Reply:
I am not going to lie I really don't understand what you mean here. `/obj/item = 'path'` is a generic that would play if nothing else is being specified,`/obj/item/screwdriver = 'path'` would play a sound for the screwdriver plus all of its subtypes unless overridden. What is wrong with this implementation?

Anyways I have updated to code to make sure everything with a list has a generic

Reply:
ah that seems easy enough to do, I'll try my best.